tests/test_definition.py

```diff
@@ -64,6 +64,27 @@ def test_definition_from_general_config():
         clean_up_external_repos(obs.config.repositories)
 
 
+def test_only_definition_from_general_config():
```

Comment:
Maybe merge with the test above (as two alternative cases) to reduce duplication?

Reply:
Good point, done.